sdk/textanalytics/tests.yml

```diff
@@ -9,4 +9,8 @@ extends:
         SubscriptionConfiguration: $(sub-config-azure-cloud-test-resources)
       Canary:
         SubscriptionConfiguration: $(sub-config-azure-cloud-test-resources)
-        Location: 'centraluseuap'
\ No newline at end of file
+        Location: 'centraluseuap'
+      UsGov:
+        SubscriptionConfiguration: $(sub-config-gov-test-resources-test)
+    Clouds: 'Public,Canary,UsGov'
```

Comment:
I think some of the TA features are not enabled in UsGov. If the Healthcare tests start failing, then we shouldn't enable this could yet

Reply:
We can consider [SkipInCloud()] attribute